ivy/functional/frontends/torch/nn/functional/convolution_functions.py

```diff
@@ -213,22 +213,22 @@ def conv_transpose1d(
 def conv_transpose2d(
     input,
     weight,
-    bias=None,
-    stride=1,
-    padding=0,
-    output_padding=0,
-    groups=1,
-    dilation=1,
+    strides=1,
+    padding="SAME",
+    output_shape=None,
+    data_format="NHWC",
+    dilations= 1,
+    bias= None,
```

Comment:
The names, order and defaults values of the arguments should be exactly the same as the [original](https://pytorch.org/docs/stable/generated/torch.nn.functional.conv_transpose2d.html) function we are replicating.